frontends/main/src/app-pages/B2BAttachPage/B2BAttachPage.tsx

```diff
@@ -23,15 +26,34 @@ const B2BAttachPage: React.FC<B2BAttachPageProps> = ({ code }) => {
     enrollment_code: code,
   })
 
+  const { isLoading: userLoading, data: user } = useQuery({
+    ...userQueries.me(),
+    staleTime: 0,
+  })
+  const { data: mitxOnlineUser } = useMitxOnlineCurrentUser()
```

Comment:
Ugh. Somehow github didn't get my comment oops.

We discussed it in slack, but issue was that `attach` needs to invalidate the useMitxOnlineCurrentUser query.

IMO, just invalidate all mitxonline queries. Attach doesn't happen often, and invalidating all of them could help avoid some bugs